Assets/MRTK/Core/Inspectors/Profiles/BaseMixedRealityProfileInspector.cs

```diff
@@ -201,5 +202,29 @@ protected static bool IsProfileLock(BaseMixedRealityProfile profile)
         {
             return MixedRealityProjectPreferences.LockProfiles && !profile.IsCustomProfile;
         }
+
+        /// <summary>
+        /// Inspect the attributes of the provided system type to determine if a configuration profile is required.
+        /// </summary>
+        /// <param name="serviceType">The system type representing the service.</param>
+        /// <returns>
+        /// True if the service is decorated with an attribute indicating a profile is required, false otherwise.
+        /// </returns>
+        protected static bool IsProfileRequired(SystemType serviceType)
+        {
+            // Services marked with the MixedRealityExtensionServiceAttribute (or a derivative)
+            // support specifying whether or not a profile is required.
+            return IsProfileRequired(serviceType?.Type);
+        }
+
+        protected static bool IsProfileRequired(Type serviceType)
+        {
+            if(serviceType != null)
+            {
+                MixedRealityExtensionServiceAttribute attribute = (serviceType != null) ? MixedRealityExtensionServiceAttribute.Find(serviceType) : null;
```

Comment:
Looks like we're null checking `serviceType` twice here. Could this be simplified a bit?